Is this action missing some required dependency for Cypress 3.8.2?

I'm getting the ["This is usually caused by a missing library or dependency."](https://github.com/lourenci/react-kanban/runs/401989842) error after upgrading Cypress to 3.8.2. 

Is this action not supporting the new Cypress versions?

Thanks

Also happens on for cypress 3.7.0 

```
2020-01-22T09:02:06.4045023Z > cypress@3.7.0 postinstall /home/runner/work/integration-tests/integration-tests/node_modules/cypress
2020-01-22T09:02:06.4045809Z > node index.js --exec install
2020-01-22T09:02:06.4046181Z 
2020-01-22T09:02:06.8537294Z Note: Overriding Cypress cache directory to: /home/runner/.cache/Cypress
2020-01-22T09:02:06.8537909Z 
2020-01-22T09:02:06.8538304Z       Previous installs of Cypress may not be found.
2020-01-22T09:02:06.8541706Z 
2020-01-22T09:02:06.8690879Z Installing Cypress (version: 3.7.0)
2020-01-22T09:02:06.8691312Z 
2020-01-22T09:02:06.8747582Z [?25l[09:02:06]  Downloading Cypress     [started]
2020-01-22T09:02:08.8600985Z [09:02:08]  Downloading Cypress     [completed]
2020-01-22T09:02:08.8611976Z [09:02:08]  Unzipping Cypress       [started]
2020-01-22T09:02:58.4964445Z [09:02:58]  Unzipping Cypress       [completed]
2020-01-22T09:02:58.4972486Z [09:02:58]  Finishing Installation  [started]
2020-01-22T09:02:58.6797214Z [09:02:58]  Finishing Installation  [completed]
2020-01-22T09:02:59.6819482Z [?25h
2020-01-22T09:02:59.6821381Z You can now open Cypress by running: node_modules/.bin/cypress open
2020-01-22T09:02:59.6821883Z 
2020-01-22T09:02:59.6822606Z https://on.cypress.io/installing-cypress
2020-01-22T09:02:59.6822738Z 
2020-01-22T09:02:59.7028940Z [?25h
2020-01-22T09:02:59.7029864Z > core-js@2.6.10 postinstall /home/runner/work/integration-tests/integration-tests/node_modules/core-js
2020-01-22T09:02:59.7030531Z > node postinstall || echo "ignore"
2020-01-22T09:02:59.7030853Z 
2020-01-22T09:02:59.8066683Z added 245 packages in 57.786s
2020-01-22T09:02:59.8158358Z [command]/usr/local/bin/npx cypress verify
2020-01-22T09:03:00.1640278Z It looks like this is your first time using Cypress: 3.7.0
2020-01-22T09:03:00.1647481Z 
2020-01-22T09:03:00.1693733Z [?25l[09:03:00]  Verifying Cypress can run /home/runner/.cache/Cypress/3.7.0/Cypress [started]
2020-01-22T09:03:08.2469362Z [09:03:08]  Verifying Cypress can run /home/runner/.cache/Cypress/3.7.0/Cypress [failed]
2020-01-22T09:03:08.2477346Z [?25hCypress failed to start.
2020-01-22T09:03:08.2477687Z 
2020-01-22T09:03:08.2477936Z This is usually caused by a missing library or dependency.
2020-01-22T09:03:08.2478021Z 
2020-01-22T09:03:08.2478212Z The error below should indicate which dependency is missing.
2020-01-22T09:03:08.2478312Z 
2020-01-22T09:03:08.2479643Z https://on.cypress.io/required-dependencies
2020-01-22T09:03:08.2479754Z 
2020-01-22T09:03:08.2479902Z If you are using Docker, we provide containers with all required dependencies installed.
2020-01-22T09:03:08.2480012Z 
2020-01-22T09:03:08.2480384Z ----------
2020-01-22T09:03:08.2480452Z 
2020-01-22T09:03:08.2480872Z Command failed: /home/runner/.cache/Cypress/3.7.0/Cypress/Cypress --no-sandbox --smoke-test --ping=852
2020-01-22T09:03:08.2481176Z 
2020-01-22T09:03:08.2481503Z ----------
2020-01-22T09:03:08.2481587Z 
2020-01-22T09:03:08.2481907Z Platform: linux (Ubuntu Linux - 18.04)
2020-01-22T09:03:08.2482042Z Cypress Version: 3.7.0
2020-01-22T09:03:08.2553177Z [?25hError: The process '/usr/local/bin/npx' failed with exit code 1
2020-01-22T09:03:08.2554085Z     at ExecState._setResult (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:894:25)
2020-01-22T09:03:08.2556998Z     at ExecState.CheckComplete (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:877:18)
2020-01-22T09:03:08.2557856Z     at ChildProcess.<anonymous> (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:777:27)
2020-01-22T09:03:08.2558308Z     at ChildProcess.emit (events.js:210:5)
2020-01-22T09:03:08.2558661Z     at maybeClose (internal/child_process.js:1021:16)
2020-01-22T09:03:08.2559016Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
2020-01-22T09:03:08.2561785Z ##[error]The process '/usr/local/bin/npx' failed with exit code 1
2020-01-22T09:03:08.2586830Z ##[error]Node run failed with exit code 1
2020-01-22T09:03:08.2708796Z ##[group]Run actions/upload-artifact@v1
2020-01-22T09:03:08.2708969Z with:
2020-01-22T09:03:08.2709467Z   name: cypress-screenshots
2020-01-22T09:03:08.2709611Z   path: cypress/screenshots
2020-01-22T09:03:08.2709739Z env:
2020-01-22T09:03:08.2709864Z   CI: 1
2020-01-22T09:03:08.2709999Z   CYPRESS_CACHE_FOLDER: /home/runner/.cache/Cypress
2020-01-22T09:03:08.2710145Z   npm_config_cache: /home/runner/.npm
2020-01-22T09:03:08.2710288Z ##[endgroup]
2020-01-22T09:03:08.5055698Z ##[error]Path does not exist /home/runner/work/integration-tests/integration-tests/cypress/screenshots
2020-01-22T09:03:08.5419112Z ##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.164.0/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.
2020-01-22T09:03:08.5590047Z ##[group]Run actions/upload-artifact@v1
2020-01-22T09:03:08.5590227Z with:
2020-01-22T09:03:08.5590366Z   name: cypress-videos
2020-01-22T09:03:08.5590498Z   path: cypress/videos
2020-01-22T09:03:08.5590626Z env:
2020-01-22T09:03:08.5590752Z   CI: 1
2020-01-22T09:03:08.5590887Z   CYPRESS_CACHE_FOLDER: /home/runner/.cache/Cypress
2020-01-22T09:03:08.5591030Z   npm_config_cache: /home/runner/.npm
2020-01-22T09:03:08.5591160Z ##[endgroup]
2020-01-22T09:03:08.8091282Z ##[error]Path does not exist /home/runner/work/integration-tests/integration-tests/cypress/videos
2020-01-22T09:03:08.8293855Z ##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.164.0/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.
2020-01-22T09:03:08.8415585Z ##[group]Run 8398a7/action-slack@v2
2020-01-22T09:03:08.8415738Z with:
2020-01-22T09:03:08.8415857Z   status: Failure
2020-01-22T09:03:08.8415973Z   author_name: github
2020-01-22T09:03:08.8416087Z   only_mention_fail: here
2020-01-22T09:03:08.8416203Z env:
2020-01-22T09:03:08.8416316Z   CI: 1
2020-01-22T09:03:08.8416421Z   CYPRESS_CACHE_FOLDER: /home/runner/.cache/Cypress
2020-01-22T09:03:08.8416552Z   npm_config_cache: /home/runner/.npm
2020-01-22T09:03:08.8417320Z   GITHUB_TOKEN: ***
2020-01-22T09:03:08.8417566Z   SLACK_WEBHOOK_URL: ***
2020-01-22T09:03:08.8417667Z ##[endgroup]
2020-01-22T09:03:09.1549064Z (node:3204) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
2020-01-22T09:03:09.2686427Z Cleaning up orphan processes
```

--- a/README.md
+++ b/README.md
@@ -11,7 +11,7 @@ name: End-to-end tests
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -30,7 +30,7 @@ name: E2E on Chrome
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     # let's make sure our tests pass on Chrome browser
     name: E2E on Chrome
     steps:
@@ -49,7 +49,7 @@ name: Chrome headless
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - uses: actions/checkout@v1
       - uses: cypress-io/github-action@v1
@@ -67,7 +67,7 @@ name: E2E in custom container
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     # Cypress Docker image with Chrome v78
     # and Firefox v70 pre-installed
     container: cypress/browsers:node12.13.0-chrome78-ff70
@@ -87,7 +87,7 @@ name: Cypress tests
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -108,7 +108,7 @@ on: [push]
 jobs:
   cypress-run:
     name: Cypress run
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -129,7 +129,7 @@ on: [push]
 jobs:
   cypress-run:
     name: Cypress run
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -152,7 +152,7 @@ name: Artifacts
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     name: Artifacts
     steps:
       - uses: actions/checkout@v1
@@ -186,7 +186,7 @@ on: [push]
 jobs:
   cypress-run:
     name: Cypress run
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -207,7 +207,7 @@ on: [push]
 jobs:
   cypress-run:
     name: Cypress run
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -230,7 +230,7 @@ on: [push]
 jobs:
   test:
     name: Cypress run
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     strategy:
       matrix:
         # run 3 copies of the current job in parallel
@@ -265,7 +265,7 @@ name: Build
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -284,7 +284,7 @@ name: With server
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -301,7 +301,7 @@ name: With server
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -323,7 +323,7 @@ name: After server responds
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -355,7 +355,7 @@ name: Visual
 on: [push]
 jobs:
   e2e:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - name: Checkout
         uses: actions/checkout@v1
@@ -381,7 +381,7 @@ name: Parallel
 on: [push]
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     strategy:
       matrix:
         # run 3 copies of the current job in parallel
@@ -420,7 +420,7 @@ You can specify the `e2e` working directory when running Cypress tests using `wo
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - uses: actions/checkout@v1
       - uses: cypress-io/github-action@v1
@@ -456,7 +456,7 @@ name: E2E
 on: push
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - uses: actions/checkout@master
       - name: Install root dependencies
@@ -483,7 +483,7 @@ name: End-to-end tests
 on: [push]
 jobs:
   cypress-run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     # let's make sure our "app" works on several versions of Node
     strategy:
       matrix:
@@ -517,7 +517,7 @@ name: E2E
 on: push
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-16.04
     steps:
       - uses: actions/checkout@master
       - name: Install dependencies

